units/spawnhost_modify.go

```diff
@@ -26,12 +66,13 @@ func init() {
 }
 
 type spawnhostModifyJob struct {
-	HostID        string                 `bson:"host_id" json:"host_id" yaml:"host_id"`
-	ModifyOptions host.HostModifyOptions `bson:"modify_options" json:"modify_options" yaml:"modify_options"`
-	job.Base      `bson:"job_base" json:"job_base" yaml:"job_base"`
+	// TODO (EVG-16066): remove HostID.
+	HostID                string                 `bson:"host_id" json:"host_id" yaml:"host_id"`
+	ModifyOptions         host.HostModifyOptions `bson:"modify_options" json:"modify_options" yaml:"modify_options"`
+	CloudHostModification `bson:"base_spawn_host_modification" json:"base_spawn_host_modification" yaml:"base_spawn_host_modification"`
```

Comment:
Preference for having the bson/json tags match the variable name

Reply:
Ah yeah, nice catch. This is what I get for waffling on naming things too many times. Fixed.